src/fides/core/export.py

```diff
@@ -363,6 +370,7 @@ def generate_system_records(  # pylint: disable=too-many-nested-blocks, too-many
                     ]
                     for category in data_categories
```

Comment:
@adamsachs do we know if this is limited to these taxonomy types? Deletes have never been something we handled fully so am curious if the same should apply for data category, etc.

Reply:
a very good question! i thought about this, but the reason we don't need to run any checks here is because there's no actual "lookup" of the data category object, the string value(s) referenced by the declaration are just taken at face value

i suppose we _could_ just run a query against the db to see if it exists and skip the row if it doesn't. but it's not like we're actually using anything in the data category, and we're sort of punting on more "proper" handling of taxonomy deletion generally, so my inclination here is just to leave it be, since we won't error.

Reply:
fair point, thanks! Ya let's leave it and see if we can prioritize some of the work we have been punting on to stop this cycle we keep finding ourselves in 🌀